api/Document.json

```diff
@@ -3370,55 +3370,6 @@
           }
         }
       },
-      "dragexit_event": {
```

Comment:
Why should we not keep this entry, and have it be in sync with the `ondragexit` entry? @ddbeck is there a guidelines for when to have each type of entry, and on what interfaces they should be?

Reply:
I'm not quite sure what it means that `onrdagexit` is supported, but the `dragexit` _event_ is not. @sideshowbarker can you elaborate on this?

My instinct says that if there's an event handler then there's probably a corresponding event (unless the event handler receives a different event, or some other oddity), but I don't actually know a lot about event APIs, so I don't know if that's a helpful instinct.

Reply:
```
-       "dragexit_event": { 
```
> Why should we not keep this entry, and have it be in sync with the `ondragexit` entry?

My preference would be to delete the `ondragexit` entry too. I don’t think we have much evidence that developers are actually relying on it in any way in practice — we don’t have any evidence that they’d miss it if we dropped it.

> I'm not quite sure what it means that `onrdagexit` is supported, but the `dragexit` _event_ is not. @sideshowbarker can you elaborate on this?

I believe the `dragexit` event  is supported *in Gecko* — it’s just that we never got around to documenting it as such in BCD. But it’s not supported in any other engines, and never will be. So it’s never been part of the web platform and never will be. For those reasons I don’t think it merits us going out of our way to retroactively document it in BCD in any way.

And given all that, my personal preference would be that we completely delete the `ondragexit` data too. If we are able to get agreement about that, then I can also add that change to this PR.

Reply:
OK, I guess this is the story as I understand it:

1. The data removed in this PR (the event itself) reports `false` for all browsers. But this data is erroneous, at least for Firefox—Gecko implemented it, but BCD never reflected that.
2. The PR leaves intact data for the `ondragexit` handler. The data for that is either strange in some way which we haven't documented (browsers other than Firefox use `ondragexit` as a handler for a different event or something else), or wrong (browsers other than Firefox don't implement `ondragexit` in any way).

But, if as you say, Firefox is the only browser that implemented `ondragexit` and it's still available today (it seems to be, though I'm basing that solely on the console tab completion for `ondragexit`), then we should:

1. Have features for both `ondragexit` and the `dragexit` event
2. Populate those features with `false` values for all the other browsers. If we're able to, set the correct values for Firefox, or open a help-wanted issue if we can't be bothered right now.
3. Set the status flags to show this is non-standard, non-experimental, and deprecated.

It's a bit of a bummer—I'd be happier if Firefox removed it so we could bin the features themselves—but I think that's what we're stuck with, if a browser is actually shipping it. In terms of documentation on MDN itself, I don't think it's worth anyone's time to actually update those docs, except maybe to add deprecation notices where there are pages.

Reply:
Sorry, @sideshowbarker I realize I kinda dropped a wall of text on you here. Is this too much? As an alternative, we could pare this down to the status change only, then open an issue to address the feature data at a later time.

(or if you just hadn't got back to this yet, that's fine too)

Reply:
Per [Confluence](http://web-confluence.appspot.com/#!/catalog?releases=%5B%22Edge_18.17763_Windows_10.0%22,%22Chrome_85.0.4183.83_Windows_10.0%22,%22Firefox_80.0_Windows_10.0%22,%22Safari_14.0_OSX_10.15.6%22%5D&q=%22dragexit%22), the `ondragexit` event handler attributes are still in Firefox.

Reply:
> As an alternative, we could pare this down to the status change only

OK, I’ve amended and pushed the commit in this PR branch to only contain a change to the `standard_track` and `deprecated` values. 

> then open an issue to address the feature data at a later time.

OK, opened https://github.com/mdn/browser-compat-data/issues/6826 for that